packages/frontend/src/components/wallet/NFTBox.js

```diff
@@ -130,12 +130,22 @@ const NFTBox = ({ tokenDetails }) => {
             {
                 ownedTokensMetadata &&
                 <div className='tokens'>
-                    {ownedTokensMetadata.map(({ token_id, metadata: { mediaUrl, title } }) => {
+                    {ownedTokensMetadata.map(({ token_id, metadata: { mediaUrl, title } }, index) => {
+                        const videoProps = index === 0 ? { autoPlay: true } : {};
```

Comment:
I think `autoPlay` should be all lowercase for consistency with other video html tag props

Reply:
![image](https://user-images.githubusercontent.com/11353590/150385015-395a253f-11de-445d-b9e9-ad1176168583.png)

That's why I changed that.